src/olympia/zadmin/forms.py

```diff
@@ -239,64 +239,3 @@ class CompatForm(BaseCompatForm):
     _ratio_choices = [('%.1f' % (x / 10.0), '%.0f%%' % (x * 10))
                       for x in xrange(9, -1, -1)]
     ratio = forms.ChoiceField(choices=_ratio_choices, required=False)
-
-
-class GenerateErrorForm(happyforms.Form):
-    error = forms.ChoiceField(choices=(
-        ['zerodivisionerror', 'Zero Division Error (will email)'],
-        ['iorequesterror', 'IORequest Error (no email)'],
-        ['heka_statsd', 'Heka statsd message'],
-        ['heka_json', 'Heka JSON message'],
-        ['heka_cef', 'Heka CEF message'],
-        ['heka_sentry', 'Heka Sentry message'],
-        ['amo_cef', 'AMO CEF message']))
```

Comment:
what for, e.g to test logging and sentry?

Sentry has a command line tool that generates an error, maybe we could add something like that too if there's need.